lib/domain/model/alert.dart

```diff
@@ -21,8 +22,16 @@ class AlertFields {
   static const String repeatIntervalTimeInMinutes = 'repeatIntervalTimeInMinutes';
 }
 
+enum AlertStatus {
+  disabled, // Alert is disabled
+  dismissed, // Repeating alert which has been dismissed by the user
```

Comment:
not sure how will dismissed be used, if a notification has been sent, then the alert has triggered status? no matter if user dismiss it or not?

Reply:
When I wrote this, I was thinking about the case when the user choose to "skip" the next repeating alert